File: to-do-app/to-do-node/src/controller/routes/routesAuth.ts
import express from 'express';
import authController from '../authController';
// import mid from '../middleware/middleware'

const router = express.Router();

// router.post('/api/auth/login', mid.validateAuth, authController.login)
router.post('/api/auth/login', authController.login)
router.post('/api/auth/recoverpassword', authController.recoverPassword)
router.post('/api/auth/passwordreset', authController.resetPassword)
router.post('/api/auth/userregistration', authController.registerNewUser)
router.post('/api/auth/useractivation', authController.activateUser)
router.post('/api/auth/useractivationrequest', authController.requestUserActivation)

export default { router };

File: webstore/webstore-go/src/router/routesAuth.go
package router

import (
	"github.com/gin-gonic/gin"
	"renanlelis.github.io/portfolio/webstore-go/src/controller"
)

func ConfigAuthRoutes(r *gin.Engine) {
	r.POST("/api/auth/login", controller.Login)
	r.POST("/api/auth/recoverpassword", controller.RecoverPassword)
	r.POST("/api/auth/passwordreset", controller.ResetPassword)
	r.POST("/api/auth/userregistration", controller.RegisterNewUser)
	r.POST("/api/auth/useractivation", controller.ActivateUser)
	r.POST("/api/auth/useractivationrequest", controller.RequestUserActivation)
}
